test/test-network/conf/25-sriov.network

```diff
@@ -3,6 +3,7 @@ Name=eni99np1
 
```

Comment:
Hmm, justification?

Reply:
Just a tiny speed up fro the test. Mentioned in the comment.